### DISCUSSION TITLE
Asembly crescendo and two volume pedals in GO in Win10.

### DISCUSSION THREAD

**Discussion by jerole8231** [GrandOrgue/grandorgue#842] [Help]
How to join this pedals to GO. Standard way give me no result. Signals from pedals are good. I've check it in MidiOx program, and for example I've joined it instead of fourth table keys Celesta. When I moved pedal then I've got a change of tonnes. It souds like a Havaiian guitar. Three pedals are connectet to separated three canals 6, 7, 8. I think that must be change a variable in Windows or GO, but it is only my suggestion? Look at the attached film. Horizontal bar is joining directly to crescendo pedal. I can't show it because of size of max 10MB of attached file.

https://user-images.githubusercontent.com/93871671/140654651-5a71d107-8245-4aa0-931f-cbcd0c4a5b2d.mp4

.

**Comment by oleg68** [comment]
You should configure the pedal in GO with a certain organ and save the organ settings.

1. Open your organ
2. Right click on the pedal image on the main organ panel
3. Click on "Listen to"
4. Roll the pedal
5. Press Ok
6. Check the pedal works as expected
7. Repeat the steps 2-6 for the other pedals
8. File->Save
9. Repeat the steps 1-8 for every organ

**Reply by jerole8231** [reply]
Hi.
I use Win 10
I right click on the pedal image and I've got card see attached file.
I've click on Listen for event.
Roll the pedal in my organ.
card changing only in field Channel - it shows number 6 (it is channel hardwer save in my processor)
Click OK. And pedal is still disconnect.
Open the card again, and it is clear wih out any changes.
,When I change fild Event on NPRN or, and the other, the card can save settings but pedal is still disconnected.
This same I've got in other computer with Win7.
May be nobody used this function in windows yet???

![Karta](https://user-images.githubusercontent.com/93871671/141185802-71eabd69-784f-49da-bc83-c46b68b64719.jpg)


I've made mistakes. Field Event-No and Device are always changing. See attaching new file. It is before using OK with out changing any fields. After OK and open again this card it is totaly empty.
![Przyc1](https://user-images.githubusercontent.com/93871671/141188205-1373808f-7950-466f-989a-e5a07b9a4686.jpg)

**Comment by midimusic** [comment]
I can confirm that this method does work for Windows 10. I have just checked it on GO v3.2.0-1.
Do not try to anticipate any settings by entering your own data - swell pedals do not usually emit NRPN midi events (but it is not impossible).
Just right click on your pedal and immediately select listen for event. Move the pedal and the data will be entered automatically. Then select OK.
csw900
